mail_connector_queue/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Mail Connector Queue
+====================
+
+This module uses the Connector Queue to send any emails generated by Odoo,
+by creating a queue job for each one.
```

Comment:
Can you please put this in readme/DESCRIPTION.rst according to https://github.com/OCA/maintainer-tools/pull/339

Reply:
A word about why this is useful would be nice too. Such as "This enables, for instance, running high volume mass mailing with low priority, to avoid blocking regular notifications.".